hypershift-operator/controllers/platform/aws/controller.go

```diff
@@ -302,15 +302,15 @@ func (r *AWSEndpointServiceReconciler) reconcileAWSEndpointServiceStatus(ctx con
 	hasPrivateRouterEPService := false
 	hasPrivateIngressControllerEPService := false
 	for _, eps := range endpointServices.Items {
-		// If a private-router AWSEndpointService exists, it means that
+		// If a router AWSEndpointService exists, it means that
```

Comment:
it means that ?

Reply:
No idea, I had just deleted the reference to private, whoever originally created the comment forgot to finish the sentence :upside_down_face: